Created GitHub markdown Cheat-sheet (#185)
o3de/community: process-docs-and-markdown/github-markdown.md

```

I think you should use an `<addr>` element here instead.

Markdown coverts text with four leading spaces
into a code block; with GFM you can wrap your code
with 

## Emoji

Look up emoji codes at [emoji-cheat-sheet.com](http://emoji-cheat-sheet.com/)

```markdown
:+1: :sparkles: :camel: :tada: :rocket: :metal:
```

:+1: :sparkles: :camel: :tada: :rocket: :metal:

## Username @ mentions
Typing an @ symbol, followed by a username, will
notify that person to come and view the comment.
This is called an “@mention”, because you’re
mentioning the individual. You can also @mention
teams within an organization

## Issue References
Any number that refers to an Issue or Pull Request
will be automatically converted into a link. Note this issue has to be located from within the repository that you are doing your work in. 
#1

## Hotkey

<kbd>⌘F</kbd>

<kbd>⇧⌘F</kbd>

```markdown
<kbd>⌘F</kbd>
``` 

Hotkey list:

| Key | Symbol |
| --- | --- |
| Option | ⌥ |
| Control | ⌃ |
| Command | ⌘ |
| Shift | ⇧ |
| Caps Lock | ⇪ |
| Tab | ⇥ |
| Esc | ⎋ |
| Power | ⌽ |
| Return | ↩ |
| Delete | ⌫ |
| Up | ↑ |
| Down | ↓ |
| Left | ← |
| Right | → |

## Footnotes

> Footnotes aren't part of the core Markdown spec, but they supported by GFM.

Reviewer: ```suggestion
Footnotes aren't part of the core Markdown spec, but they are supported on GitHub.
```

```Here is a simple footnote[^1].

A footnote can also have multiple lines[^2].  

You can also use words, to fit your writing style more closely[^note].

[^1]: My reference.
[^2]: Every new line should be prefixed with 2 spaces.  
  This allows you to have a footnote with multiple lines.
[^note]:
    Named footnotes will still render with numbers instead of the text but allow easier identification and linking.  
    This footnote also has been made with a different syntax using 4 spaces for new lines.
```

Reners to: 

![footnote](/process-docs-and-markdown/files/media/footer.png)

## YouTube Videos

> They can't be added directly but you can add an image with a link to the video like this:

Reviewer: ```suggestion
Youtube videos can't be embedded directly but you can add an image with a link to the video like this:
```

```markdown
<a href="http://www.youtube.com/watch?feature=player_embedded&v=YOUTUBE_VIDEO_ID_HERE
" target="_blank"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg" 
alt="IMAGE ALT TEXT HERE" width="240" height="180" border="10" /></a>
```

<a href="http://www.youtube.com/watch?feature=player_embedded&v=YOUTUBE_VIDEO_ID_HERE
" target="_blank"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg" 
alt="IMAGE ALT TEXT HERE" width="240" height="180" border="10" /></a>

> Or, in pure Markdown, but losing the image sizing and border:

Reviewer: ```suggestion
Or, in pure Markdown, but without the image sizing and border:
```
